.github/workflows/release_climt.yml

```diff
@@ -31,7 +31,21 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           path: ./wheelhouse/*.whl
+      
+      - name: Build on Linux
+        env:
+          CIBW_BUILD: cp36-manylinux_x86_64 cp37-manylinux_x86_64 cp38-manylinux_x86_64
+        run: |
+          if: ${{ runner.os == 'Linux'  }}
```

Comment:
The if statement is a top-level statement, at the same level as `run:`. see how it is used in `windows_package.yml`

Reply:
ok